src/GeoPing.FrontEnd/src/pages/adminAllChecklistPage.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
```

Comment:
Please separate pages for public users and admins in different directories. to keep them consistent without prefixes.